features/adb-openshift.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Command output from various OpenShift related commands in ADB
+  service-manager should return the correct output from commands affecting OpenShift in ADB
+
+  @ci-only
+  Scenario Outline: Boot and execute commands
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    begin
+      require 'vagrant-libvirt'
+    rescue LoadError
+      # NOOP
+    end
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker, openshift'
+
+      config.servicemanager.openshift_image_tag = 'v1.2.1'
```

Comment:
@hferentschik I think we can remove these three lines as the latest tags are now handled as part of PR and it should able to correctly fetch the version for latest bits.

This will be kind of test for regex change I made to fetch the correct latest stable version.